packit_service/worker/parser.py

```diff
@@ -1292,3 +1295,25 @@ def parse_pagure_pull_request_comment_event(
             comment=comment,
             comment_id=comment_id,
         )
+
+    @staticmethod
+    def parse_new_hotness_update_event(event) -> Optional[NewHotnessUpdateEvent]:
+        if "hotness.update.bug.file" not in event.get("topic", ""):
+            return None
+
+        package_name = event.get("package")
```

Comment:
Shouldn't we use `package_name` from a Fedora subdictionary instead? Or provide both. This one is probably what we call `upstream_name`.

Reply:
I also wasn't sure and therefore checked the code how 'package' is set [here](https://github.com/fedora-infra/the-new-hotness/blob/363acd33623dadd5fc3b60a83a528926c7c21fc1/hotness/hotness_consumer.py#L385) and [here](https://github.com/fedora-infra/the-new-hotness/blob/363acd33623dadd5fc3b60a83a528926c7c21fc1/hotness/hotness_consumer.py#L444-L455) so it should be the Fedora package name...but I will double check it

Reply:
Ok, we can easily fix this if it is not true. (Could you please add  a comment about this to the parser so we don't forget why it's like that?)